skorch/net.py

```diff
@@ -156,7 +156,9 @@ class NeuralNet:
     device : str, torch.device (default='cpu')
       The compute device to be used. If set to 'cuda', data in torch
       tensors will be pushed to cuda tensors before being sent to the
-      module.
+      module. If set to 'auto', computer device will select 'cuda' if
+      available, otherwise it will use the default. If set to None, then
```

Comment:
```suggestion
      module. If set to 'auto', 'cuda' will be selected if available,
      otherwise the default will be used. If set to None, then
```